### DISCUSSION TITLE
Comprehensive Video Tutorial - Short Course

### DISCUSSION THREAD

**Discussion by volfadar** [medusajs/medusa#6144] [Feature Requests]
I really appreciate Medusa JS with all the features and flexibility it offers, which greatly facilitates developing ecommerce websites - our daily work at the studio for our clients. 

However, I feel Medusa could improve its documentation by adding more visual and audio components, not just text-based docs. Because not everyone learns effectively by reading. 

For me it's no problem, but it becomes a little difficult when I want to convey this to my teams who have different learning preferences, especially since we don't come from native English-speaking countries. 

ps. it would be great if Medusa could endorse influencers like the Josh Tried Coding channel to create Medusa JS tutorials for building sites from scratch and getting them online. 

![Screenshot_80](https://github.com/medusajs/medusa/assets/44869940/643d7e6f-3cd6-47eb-8603-eda3e7d47ab4)
His **[Payload CMS tutorial](https://www.youtube.com/watch?v=06g6YJ6JCJU)** was incredibly helpful. By clearly demonstrating the platform and key concepts, it greatly reduced our team's training time

**Comment by srindom** [comment]
Hi @volfadar - thank you very much for the input! Curious to hear if you have good ideas about how to outline a tutorial like that? What topic would you cover first, and where would you go from there? :)

**Comment by DevHanza** [comment]
Here is something I found on youtube: 

Medusa Development Course:
https://www.youtube.com/playlist?list=PLb5Hb5lLAR1kbhxaHifGdW-EOL-LyfLiK

**Reply by AKRking** [reply]
I watched it,it felt advanced for beginners, couldn't continue
